other/materials_designer/workflows/band_gap.ipynb

```
@@ -100,6 +100,26 @@
    "metadata": {},
    "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"API_HOST\"] = \"localhost\"\n",
+    "os.environ[\"API_PORT\"] = \"3000\"\n",
+    "os.environ[\"API_SECURE\"] = \"false\"\n",
+    "\n",
+    "\n",
+    "API_HOST = \"localhost\"\n",
+    "API_PORT = \"3000\"\n",
```

Ruff (0.15.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)